Meadowlark-js/services/meadowlark-fastify/readme.md

```diff
@@ -11,15 +11,21 @@ Instructions for running a local "developer" environment:
    * `DOCUMENT_STORE_PLUGIN` - The backend document store that Meadowlark will use
      * If using MongoDB set the value to `@edfi/meadowlark-mongodb-backend`
        * Update `MONGO_URI` to the location of your Mongo Instance
+         * According to the configuration indicated in the documentation, the default value for mongo credentials is
+        `mongodb://admin:admin@mongo1:27017,mongo2:27018,mongo3:27019/?replicaSet=rs0`
      * If using PostgreSQL set the value to `@edfi/meadowlark-postgresql-backend`
        * Update `POSTGRES_USERNAME` and `POSTGRES_PASSWORD`
        * If your PostgreSQL instance is not running on the default port (5432), you can set `POSTGRES_PORT` to
          the port PostgreSQL is using
    * `QUERY_HANDLER_PLUGIN` and `LISTENER1_PLUGIN` - Uncomment these two for GET query support using OpenSearch
    * `AUTHORIZATION_STORE_PLUGIN` This is the plugin for the authorization store, it should be set to a NPM package name
      * To use the Meadowlark-js provided Authorization plugin, this should be set to `@edfi/meadowlark-mongodb-backend`
+   * OAUTH_SIGNING_KEY - The OAUTH_SIGNING_KEY may need quotation marks around the value, unlike other keys.
```

Comment:
I don't think I've ever used quotation marks around this. Shouldn't hurt of course